ext/ddtrace_profiling_native_extension/collectors_discrete_dynamic_sampler.c

```diff
@@ -310,6 +290,7 @@ VALUE discrete_dynamic_sampler_state_snapshot(discrete_dynamic_sampler *sampler)
 // Below here is boilerplate to expose the above code to Ruby so that we can test it with RSpec as usual.
 
 static VALUE _native_new(VALUE klass);
+static VALUE _native_initialize(VALUE klass, VALUE now);
```

Comment:
Minor: The argument here is actually the instance ;)

```suggestion
static VALUE _native_initialize(VALUE self, VALUE now);
```